server/galaxyls/tests/files/simple_conditional_01_test.xml

```diff
@@ -1,35 +1,15 @@
 <test expect_num_outputs="$1">
     <!--TODO: auto-generated test case. Please fill in the required values-->
-    <param name="param-01" value="${2|true,false|}"/>
     <conditional name="c1">
         <param name="c1.action" value="a1"/>
-        <param name="c1.a1.p1" value="$3"/>
+        <param name="c1.a1.p1" value="$2"/>
     </conditional>
-    <output name="sample">
```

Comment:
@davelopez why have you excluded the outputs? They are important for a test.

Reply:
Yes, this is intentional, I will add them in a separate file to keep them as `simple` tests within smaller scope. And also add other `complex` tests combining multiple kinds of inputs and outputs.